config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Policy.java

```diff
@@ -57,7 +59,8 @@
     @JsonIgnore
     private AnnotatedEntity<? extends GatewayEntity> annotatedEntity;
 
-    public Policy() {}
+    public Policy() {
```

Comment:
This file change isn't required. You can revert this file.

Reply:
OK